samples/cooperative-cancellation/Core_9/Server/LongRunningMessageHandler.cs

```diff
@@ -12,11 +13,21 @@ public async Task Handle(LongRunningMessage message, IMessageHandlerContext cont
     {
         log.Info($"Received message {message.DataId}. Entering loop.");
 
-        do
+        // The try-catch block is only for the purpose of demonstrating the sample.
+        try
+        {
+            while (true)
+            {
+                log.Info("Press any key to cancel the 3 second delay loop and stop the endpoint.");
```

Comment:
In my opinion, I'd take out the try-catch, and change this message to "Handler still running. Press any key to forcibly stop the endpoint."

Reply:
Yeah, I agree with that.  I was a little hesitant including the error handling in the handler since we explicitly state customers shouldn't do that.  I think you're right, it's better to just illustrate the proper way to implement it.  I'll make the appropriate changes.